script/experiments/export_experiment_monitoring_data.py

```diff
@@ -223,7 +223,7 @@ def export_dataset(
             source_project,
             destination_project,
         )
-        p.map(_experiment_data_export, active_experiments)
+        p.map_async(_experiment_data_export, active_experiments).get(600)   # timeout after 10 minutes
```

Comment:
Using `map_async` in combination with [`get`](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Queue.get), a timeout can be set. So if these task do not finish after 10 minutes (usually they finish within 1 minute) then an exception will be thrown.